examples/CIB/ex1/main.cpp

```diff
@@ -379,8 +430,53 @@ main(int argc, char* argv[])
                 navier_stokes_integrator->setStokesSolverNeedsInit();
             }
 
+            int coarsest_ln = 0;
+            int finest_ln = patch_hierarchy->getFinestLevelNumber();
+
+            // Update the location of the box for time n + 1
+            hydro_force->updateStructureDomain(0, dt, Eigen::Vector3d::Zero(), patch_hierarchy);
+            hydro_force->updateStructureDomain(1, dt, Eigen::Vector3d::Zero(), patch_hierarchy);
+
```

Comment:
The next chunk of code should be done inside the force evaluator object itself to reduce boilerplate code.